pkg/network/encoding/encoding_test.go

```diff
@@ -386,10 +419,10 @@ func TestSerialization(t *testing.T) {
 
 func TestFormatHTTPStatsByPath(t *testing.T) {
 	var httpReqStats http.RequestStats
-	httpReqStats.AddRequest(100, 12.5)
-	httpReqStats.AddRequest(100, 12.5)
-	httpReqStats.AddRequest(405, 3.5)
-	httpReqStats.AddRequest(405, 3.5)
+	httpReqStats.AddRequest(100, 12.5, 0)
+	httpReqStats.AddRequest(100, 12.5, 1)
+	httpReqStats.AddRequest(405, 3.5, 2)
+	httpReqStats.AddRequest(405, 3.5, 4)
```

Comment:
Update to use userspace constants after addressing comment in `pkg/network/ebpf/kprobe_types.go`

Reply:
not sure it's a good idea to have a type for tag values and expose it to the userspace.
As we need to define the tags name for all platform.

Here an minimal implementation https://github.com/DataDog/datadog-agent/pull/9171/commits/9e107910e5b8268afed0b48affe65593267006a1